boards/st/b_u585i_iot02a/b_u585i_iot02a_stm32u585xx_ext_flash_app.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: b_u585i_iot02a/stm32u585xx/xip
+name: ST B_U585I_IOT02A Discovery kit xip target
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 786
+flash: 512
```

Comment:
416? (`slot0_partition` and `slot1_partition` sizes)

Reply:
`flash:` tag allows twister to know if the board can be used for a samples/tests for zephyr image size contraints.
It should reflect the size of the chosen `zephyr,flash` partition IUUC.

Reply:
updated to "416"